Update "How Moderation Works" (#287)
RetroAchievements/docs: docs/guidelines/users/code-of-conduct.md

## How Moderation Works

> If you feel moderation is needed, ping the @moderator role on our Discord server, or direct your issues to RAdmin, a shared mod-mail account. If you prefer anonymity, reaching out to an active moderator privately is also an option.

Reviewer: ```suggestion
If you feel moderation is needed, ping the @Moderator role on our Discord server, or send an on-site DM to [RAdmin](https://retroachievements.org/messages/create?to=RAdmin), a shared mod-mail account. If you prefer anonymity, reaching out to an active moderator privately is also an option.
```

> The moderation goal is to assure all users adhere to the User's Code of Conduct. While this Code of Conduct is comprehensive, our moderators reserve the right to take action against violations that threaten a safe- and healthy community.

Reviewer: ```suggestion
The moderation team's goal is to ensure all users adhere to the [User's Code of Conduct](/guidelines/users/code-of-conduct). While this Code of Conduct is comprehensive, our moderators reserve the right to take action against violations that threaten a safe, healthy, and inclusive community.
```

> Our intention is always to resolve issues diplomatically, asking users to take a step back or remind them of the rules. If this is ignored, additional action will be taken that will be applied to your Discord account, as well as your RetroAchievements account.

Reviewer: ```suggestion
We prioritize resolving issues diplomatically by first asking users to step back and reminding them of our community rules. However, if these warnings are ignored, we will take enforcement action against both your Discord and RetroAchievements accounts.
```

> - If the violation is low, moderators may opt to mute the user, restricting all their social capabilities on both the Discord server and the site. This will not affect anything related to earning achievements. Likewise, you will be able to contact team accounts during this mute duration.

Reviewer: ```suggestion
- If the violation is minor, moderators may opt to mute the user, restricting all their social capabilities on both the Discord server and the site. This does not affect anything related to earning achievements. Likewise, the user will be able to contact team accounts during this mute duration.
```

>    - Note that once a (junior) developer breaks the Code of Conduct and receives a mute, this will double as a developer ban at the discretion of Developer Compliance or the Code Reviewer team. Moderators will inform the teams when this happens.

Reviewer: ```suggestion
   - Note that if a Developer or Junior Developer breaks the Code of Conduct and receives a mute, the mute also doubles as a developer suspension at the discretion of Developer Compliance or the Code Reviewer team. Moderators will inform the teams when this happens.
```

- Severe violations will result in a ban on your account, removing you from the Discord server and making your RetroAchievements profile inaccessible.

> Note: Violations that result in mutes are shared between the Discord server and RetroAchievements site. Likewise, a ban on our site will automatically result in a ban on Discord. A ban from the Discord may lead to a RetroAchievement site ban or other restrictive action taken, depending on the severity. 

Reviewer: ```suggestion
Note: Violations that result in mutes are shared between the Discord server and RetroAchievements site. Likewise, a ban on our site will automatically result in a ban on Discord. A ban from the Discord may lead to a RetroAchievements site ban or other restrictive action taken, depending on the severity. 
```

> If you've been banned and want to appeal to have your access back, send an email to <ramodmail@gmail.com> or a Direct Message to [RAdmin](http://retroachievements.org/createmessage.php?t=RAdmin).

Reviewer: ```suggestion
If you've been banned and want to appeal to have your account restored, send an email to <ramodmail@gmail.com> or a Direct Message to [RAdmin](https://retroachievements.org/messages/create?to=RAdmin).
```
